### PR TITLE
chore: cleanup pkg/config globals

### PR DESCRIPTION
**Pull request by maryamtahhan**
Cleanup the global variables in pkg/config

### CODE REVIEW

**Comment by maryamtahhan** [comment]
Double check if we need to introduce locking

**Comment by maryamtahhan** [comment]
> Double check if we need to introduce locking

It looks like the set functions are only called in one place. all the rest are gets - so this should be fine without locking.

**Comment by rootfs** [comment]
@sthaha @sunya-ch can you take a look too? thanks

File: pkg/collector/stats/node_stats.go [hunk 1]
```diff
@@ -25,24 +25,38 @@ import (
...
 type NodeStats struct {
 	Stats
 
 	// IdleResUtilization is used to determine idle pmap[string]eriods
 	IdleResUtilization map[string]uint64
 }
 
+// NodeName returns the node name
+func NodeName() string {
+	return GetNodeName()
```
Comment (sthaha) [inline pkg/collector/stats/node_stats.go:37]: How about we rename GetNodeName() to NodeName() ?

File: pkg/bpf/test_utils.go [hunk 2]
```diff
@@ -11,6 +11,7 @@ type mockExporter struct {
 }
 
 func DefaultSupportedMetrics() SupportedMetrics {
+	config.GetConfig()
```
Comment (sthaha) [inline pkg/bpf/test_utils.go:14]: Is this needed here?

**Comment by maryamtahhan** [comment]
Hi @sthaha 
I incorporated the suggested changes. Please let me know if there's anything else.

Thanks
Maryam

**Review by sthaha — APPROVED** [review]

**Pull request merged**